internal/capsule/docker/engine.go

```diff
@@ -0,0 +1,138 @@
+package docker
+
+import (
+	"context"
+	"io"
+	"time"
+
+	"github.com/caesium-dev/caesium/internal/capsule"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/docker/docker/client"
+)
+
+// Engine defines the interface for treating the
+// Docker API as a capsule.Engine.
+type Engine interface {
+	capsule.Engine
+}
+
```

Comment:
What's the point of doing this?

Reply:
In the event that we want to add docker specific methods, you could cast a `capsule.Engine` to a `docker.Engine` and use them.